src/bin/cql-stress-cassandra-stress/java_generate/distribution/enumerated.rs

```diff
@@ -0,0 +1,36 @@
+use anyhow::Result;
+use rand_distr::{Distribution, WeightedIndex};
+
+#[derive(Clone)]
+pub struct EnumeratedDistribution<T: Copy + std::fmt::Display> {
+    items: Vec<(T, f64)>,
+    dist: WeightedIndex<f64>,
+}
+
+impl<T: Copy + std::fmt::Display> EnumeratedDistribution<T> {
+    pub fn new(items: &[(T, f64)]) -> Result<Self> {
```

Comment:
Nit: take `items` by value (i.e. `items: Vec<(T, f64)>`). If the caller prepared a `Vec<(T, f64)>` and no longer uses it, it will let them avoid a copy. In other cases, there should be no problem for the caller copy.